src/python/pants/help/help_info_extracter_test.py

```diff
@@ -96,6 +97,14 @@ def test_deprecated(self):
     self.assertEqual('do not use this', ohi.removal_hint)
     self.assertIsNotNone(ohi.deprecated_message)
 
+  def test_enum(self):
+    class LogLevel(Enum):
+      INFO = 'info'
+      DEBUG = 'debug'
+    kwargs = {'choices': LogLevel}
+    ohi = HelpInfoExtracter('').get_option_help_info([], kwargs)
+    self.assertEqual(', '.join(e.value for e in LogLevel), ohi.choices)
```

Comment:
Nit: we started using `assert val1 == val2` instead of `self.assertEqual`. Why? Beyond less typing, Pytest has some really nice magic that will show how each value was derived: https://docs.pytest.org/en/latest/assert.html.

Reply:
TIL and I quite like that functionality. I've just updated this one assert for now but can update the rest of the file before merging.

Reply:
It's an awesome feature I also only recently learned about! Another benefit: not having to remember so many `self.assert*` methods -- all you need is `assert` and `pytest.assertRaises`